operator/data/profiles/default.yaml

```diff
@@ -143,7 +143,7 @@ spec:
 
   # Auto injection feature
     sidecarInjector:
-      enabled: true
+      enabled: false
```

Comment:
is this going to 1) turn down injector feature or 2) disable the inject deployment and service but still have the feature enabled in pilot? 

if 1), why do we turn it off in default profile?

Reply:
We want the feature "auto injection" to be enabled. We want the component, sidecarInjector to be disabled (because it is in pilot). 

trying again from master, I think its actually behaving differently now, and completely broken. Even if we have injector=enabled, we still prune it (!!), likely because of the change we made about the component name changing?

Reply:
see https://github.com/istio/istio/issues/20387

Reply:
that makes sense, somehow we may need delete sidecarInjector from IOP API in the future.